testing/kuttl/e2e/major-upgrade-missing-image/11-assert.yaml

```diff
@@ -0,0 +1,11 @@
+---
+# Since the cluster is missing the annotation, we get this condition
+apiVersion: postgres-operator.crunchydata.com/v1beta1
+kind: PGUpgrade
+metadata:
+  name: empty-image-upgrade
+status:
+  conditions:
+  - type:   "Progressing"
+    status: "False"
+    reason: "PGClusterPrimaryNotIdentified"
```

Comment:
:thinking: Wonder if we should put these `reasons`s in our docs.

Reply:
Agreed, once this is finalized.